core/src/providers/anthropic.rs

```diff
@@ -106,15 +106,29 @@ impl AnthropicLLM {
         temperature: f32,
         top_p: f32,
         stop: &Vec<String>,
-        max_tokens: Option<i32>,
+        mut max_tokens: Option<i32>,
     ) -> Result<LLMChatGeneration> {
         assert!(self.api_key.is_some());
 
         let prompt = self.chat_prompt(messages);
+
         let mut stop_tokens = stop.clone();
         stop_tokens.push(String::from("\n\nHuman:"));
         stop_tokens.push(String::from("\n\nAssistant:"));
 
+        match max_tokens {
```

Comment:
Could this be simplified as `if max_tokens.is_none() || max_tokens.unwrap() == -1) {...}` or a slightly rustier version of that ? 
Could also avoid the `mut` and assign to a new var

Reply:
Updated :+1: